functions/auth.js

```diff
@@ -227,6 +232,24 @@ exports.connectI4AOAuthHandler = functions.https.onRequest(async (req, res) => {
   res.redirect(customTokenReturnUrl.toString());
 });
 
+exports.getUsersEmailsById = functions.https.onCall(async (data, context) => {
+  checkAuth(context);
+  const { usersIds } = data;
+  const authUsersEmails = await Promise.all(
+    chunk(usersIds, 100).map(async (chunkUsers) => {
+      const chunkUserIds = chunkUsers.map((userId) => {
+        return { uid: userId };
+      });
+      const authUsersResult = await admin.auth().getUsers(chunkUserIds);
+      return authUsersResult.users.map((user) => ({
+        email: user.email || "",
+        uid: user.uid,
+      }));
+    })
+  );
```

Comment:
@ermiaSparkle no need an empty row

Reply:
@dmitrii-zolotukhin What you mean?

Reply:
fixed in 5de97cde144760c02d63d94cad7ef3f213a5d232